src/features/user/Settings/ImpersonateUser/ImpersonateUserForm.tsx

```diff
@@ -16,6 +17,7 @@ const ImpersonateUserForm = (): ReactElement => {
   const router = useRouter();
   const { t } = useTranslation('me');
   const [isInvalidEmail, setIsInvalidEmail] = useState(false);
+  const [isProcessing, setIsProcessing] = useState<boolean>(false);
```

Comment:
there is no need to define type for Boolean. typescript can implicitly detect this